public/js/selfoss-db.js

```diff
@@ -162,6 +807,64 @@ selfoss.dbOnline = {
 selfoss.db = {
 
 
+    storage: null,
+    online: true,
+    enableOffline: Cookies.get('enableOffline') == 'true',
```

Comment:
This simple boolean is there to remember if the user wants offline before creating the needed Indexddb database. A cookie seems to be the way to go for this simple thing.

Reply:
[`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) is independent of `indexedDb`. It is also simpler than cookies and limited to client only (which is fine, since the server does not need to know the values)